### PR TITLE
[LM-11240] Peer testing

### PR DESCRIPTION
**Pull request by divyanshgandhilm**
Updating SDK with issues from peer testing, customisability testing

### CODE REVIEW

File: example/lib/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+API_KEY = b3a5e07d-85c4-4d8d-9ec0-ca07e841b35b
```
Comment (AnujLM) [inline example/lib/.env:1]: Remove api key from commit

Comment (AnujLM) [reply example/lib/.env:1]: add .env file in .gitignore file

File: example/lib/main.dart [hunk 2]
```diff
@@ -12,10 +15,108 @@ final GlobalKey<ScaffoldMessengerState> rootScaffoldMessengerKey =
...
+  debugPrint("--- Notification received in LEVEL 1 ---");
+  await LMChatNotificationHandler.instance
+      .handleNotification(message, false, rootNavigatorKey);
+}
+
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
+  setupNotifications();
   await LMChatCore.instance.initialize(
     apiKey: "b3a5e07d-85c4-4d8d-9ec0-ca07e841b35b",
```
Comment (AnujLM) [inline example/lib/main.dart:33]: remove api key from code

File: lib/packages/core/lib/src/views/home/home_configuration.dart [hunk 3]
```diff
@@ -0,0 +1 @@
+class LMChatHomeConfig {}
```
Comment (AnujLM) [inline lib/packages/core/lib/src/views/home/home_configuration.dart:1]: empty config file?

**Review by AnujLM — CHANGES_REQUESTED** [review]

**Review by TyagiLikeMinds — APPROVED** [review]

**Pull request merged**